.circleci/config.yml

```diff
@@ -45,14 +45,34 @@ jobs:
       - install-bazel-linux-rbe
       - checkout
       - run: bazel run @graknlabs_build_tools//checkstyle:test-coverage
-      - run: |
-          SONARCLOUD_CODE_ANALYSIS_CREDENTIAL=$SONARCLOUD_CREDENTIAL \
-            bazel run @graknlabs_build_tools//sonarcloud:code-analysis -- \
-            --organisation $SONARCLOUD_ORGANISATION --project-key $SONARCLOUD_PROJECT \
-            --commit-id=$CIRCLE_SHA1 --branch=$CIRCLE_BRANCH
       - run-bazel-rbe:
           command: bazel build //...
-  
+  code-analysis-pr:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run: |
+          export SONARCLOUD_CODE_ANALYSIS_CREDENTIAL=$SONARCLOUD_CREDENTIAL
+          bazel run @graknlabs_build_tools//sonarcloud:code-analysis -- \
+            --mode pr \
+            --sonar-org $SONARCLOUD_ORGANISATION --sonar-project-key $SONARCLOUD_PROJECT \
+            --github-org graknlabs --github-repo $CIRCLE_PROJECT_NAME \
+            --branch $CIRCLE_BRANCH --commit $CIRCLE_SHA1 \
+            --pr 5320 --target-branch master
+  code-analysis-master:
```

Comment:
let's call it `build-analysis` as this is the main one

Reply:
ok